Furball.Vixie.Backends.OpenGL.Shared/BufferObjectGL.cs

```diff
@@ -182,7 +183,7 @@ public static unsafe BufferObjectGL CreateNew<pDataType>(IGLBasedBackend backend
         /// Unlocks the Buffer, so that other buffers can be bound
         /// </summary>
         /// <returns>Self, used for chaining Methods</returns>
-        internal BufferObjectGL Unlock() {
+        public BufferObjectGL Unlock() {
```

Comment:
Locking is intentionally internal only, this used to/is used to prevent a Shader/Buffer/Whatever switches during which they shouldn't ever be switched